src/ui/component/DisciteModule.html

```diff
@@ -1,24 +1,43 @@
 <div class="discite__item u-clearfix discite__item--module">
-    <div class="discite__main discite__main--module">
-        <h3 class="heading bold-small heading--text">
-            {{module.title}}
-        </h3>
+    <div class="discite__main discite__main--module--{{modifier || ''}}">
```

Comment:
in BEM there shouldn't be any cases where you have a modifier modifier. I've created an element  .discite__counter. Could you put this in there? refer to `DisciteCourse.html` for structure.

Reply:
done